examples/with-next-sass/components/hello-world.module.scss

```diff
@@ -1,5 +1,8 @@
+@use '@material/typography';
```

Comment:
The `includePaths` option is not needed if you prefix it with `~` right 🤔 

```suggestion
@use '~@material/typography';
```

That opts in the file for module resolution through webpack

Reply:
Nope, because the Sass modules that I'm using have their own imports (that do not use the `~` in their `@use` statements.

I don't want to mess with my dependencies source code.

Reply:
Seems like those modules are not working in the general ecosystem overall then?